lib/js/index.ts

```diff
@@ -1,3 +1,5 @@
+import '@mdi/font/css/materialdesignicons.css';
```

Comment:
it's ~50kB gzipped CSSes and almost 300kB for woff2.

Do we really want to include that in DS